File: Java/03. Code Related to Arrays/59. Sort the elements of an array.java
import java.util.*;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.print("Enter the number of elements of the Array: ");
        int size = userInput.nextInt();
        int[] numbers = new int[size];

        for (int i = 0; i < size; i++) {
            System.out.print("Element " + (i + 1) + ": ");
            numbers[i] = userInput.nextInt();
        }
        int n = numbers.length;
        for (int i = 0; i < n; i++) {
            for (int j = i+1; j < n; j++) {
                if (numbers[i] > numbers[j]) {
                    int temp = numbers[i];
                    numbers[i] = numbers[j];
                    numbers[j] = temp;
                }
            }
        }
        System.out.println(Arrays.toString(numbers)); 
        userInput.close();
    }
}

File: Python/03. Code Related to Arrays/59. Sort the elements of an array.py
lengthOfArray = int(input("Enter the length of an Array:"))
array = []

for i in range(lengthOfArray):
    element = int(input(f"Enter {i+1} element :"))
    array.append(element)

n = len(array)
for i in range(0, n):
    for j in range(i + 1, n):
        if array[i] > array[j]:
            temp = array[i]
            array[i] = array[j]
            array[j] = temp
print(array)

# Simple Method

# print(sorted(array))
# print(sorted(array,reverse=True))